docs/csharp/programming-guide/classes-and-structs/constructors.md

```diff
@@ -27,7 +28,7 @@ If a constructor can be implemented as a single statement, you can use an [expre
 
 ## Static constructors
 
-The previous examples have all shown instance constructors, which create a new object. A class or struct can also have a static constructor, which initializes static members of the type.  Static constructors are parameterless. If you don't provide a static constructor to initialize static fields, the C# compiler will supply a default static constructor that initializes static fields to their default value as listed in the [Default Values Table](../../../csharp/language-reference/keywords/default-values-table.md). 
+The previous examples have all shown instance constructors, which create a new object. A class or struct can also have a static constructor, which initializes static members of the type.  Static constructors are parameterless. If you don't provide a static constructor to initialize static fields, the C# compiler will supply a default static constructor that initializes static fields to their default value as listed in the [Default Values Table](../../../csharp/language-reference/keywords/default-values-table.md).
```

Comment:
This is a nit, and unrelated to this PR, but since it's a paragraph that's getting updated....

The C# spec doesn't require that a static constructor is generated.  The last paragraph before the first example in the section on [static constructors](https://docs.microsoft.com/dotnet/csharp/language-reference/language-specification/classes#static-constructors) says:

> ... Each of the static fields is initialized to its default value (Default values). Next, the static field initializers (Static field initialization) are executed for those static fields. Finally, the static constructor is executed.

Following the link for [Default values](https://docs.microsoft.com/dotnet/csharp/language-reference/language-specification/variables#default-values) shows the following text, after noting that static variables are initialized using these rules:

> Initialization to default values is typically done by having the memory manager or garbage collector initialize memory to all-bits-zero before it is allocated for use. 

After that (too long winded) explanation, I have the following suggestion:

```suggestion
The previous examples have all shown instance constructors, which create a new object. A class or struct can also have a static constructor, which initializes static members of the type.  Static constructors are parameterless. If you don't provide a static constructor to initialize static fields, the C# compiler initializes static fields to their default value as listed in the [Default Values Table](../../../csharp/language-reference/keywords/default-values-table.md).
```